electrum/gui/kivy/uix/dialogs/tx_dialog.py

```diff
@@ -250,6 +253,42 @@ def _do_rbf(
         self.update()
         self.do_sign()
 
+    def do_cpfp(self):
+        from .cpfp_dialog import CPFPDialog
+        parent_tx = self.tx
+        new_tx = self.wallet.cpfp(parent_tx, 0)
+        total_size = parent_tx.estimated_size() + new_tx.estimated_size()
+        parent_txid = parent_tx.txid()
+        assert parent_txid
+        parent_fee = self.wallet.get_tx_fee(parent_txid)
+        if parent_fee is None:
+            self.app.show_error(_("Can't CPFP: unknown fee for parent transaction."))
+            return
+        cb = partial(self._do_cpfp, parent_tx=parent_tx)
+        d = CPFPDialog(self.app, parent_fee, total_size, new_tx=new_tx, callback=cb)
+        d.open()
+
+    def _do_cpfp(
+            self,
+            fee,
+            max_fee,
+            is_final,
+            *,
+            parent_tx: Transaction
```

Comment:
(nit) In case a function definition has parameters listed one per line as is done here, it's best to add a comma even after the last parameter. This ensures that if a new parameter is added later, this line will not have to be touched, and so it minimises the diff and potential merge conflicts.

```suggestion
            parent_tx: Transaction,
```